README.md

```diff
@@ -23,6 +23,11 @@ ParseXbrl.parse('./test/sampleXbrlDocuments/amazon_10k.xml').then(function(parse
 });
 ```
 
+###### Example CLI usage:
+
+```
+parse-xbrl inputfile.xbrl outputfile.json
```

Comment:
If you are pulling it down via cloning, you have to npm link it first

After publishing, everyone who "npm install -g "s it automatically has it npm linked